Sources/FCM/FCMAndroidConfig/FCMAndroidNotification.swift

```diff
@@ -1,75 +1,103 @@
-public struct FCMAndroidNotification: Codable, Equatable {
+public struct FCMAndroidNotification: Sendable, Codable, Equatable {
+    enum CodingKeys: String, CodingKey {
+        case title
+        case body
+        case icon
+        case color
+        case sound
+        case tag
+        case clickAction = "click_action"
+        case bodyLocalizationKey = "body_loc_key"
+        case bodyLocalizationArgs = "body_loc_args"
+        case titleLocalizationKey = "title_loc_key"
+        case titleLocalizationArgs = "title_loc_args"
+        case channelID = "channel_id"
+        case ticker
+        case sticky
+        case eventTime = "event_time"
+        case localOnly = "local_only"
+        case notificationPriority = "notification_priority"
+        case defaultSound = "default_sound"
+        case defaultVibrateTimings = "default_vibrate_timings"
+        case defaultLightSettings = "default_light_settings"
+        case vibrateTimings = "vibrate_timings"
+        case visibility
+        case notificationCount = "notification_count"
+        case lightSettings = "light_settings"
+        case image
+    }
+    
     /// The notification's title.
     /// If present, it will override FCMNotification.title.
-    public var title: String?
+    public let title: String?
```

Comment:
These can be mutable if needed as a struct. I don't know if anything needs to mutate them but might be useful

Reply:
Yep, probably a question for @MihaelIsaev